client/my-sites/checkout/composite-checkout/components/prepurchase-notices/akismet-product-overlaps-owned-product-notice.tsx

```diff
@@ -0,0 +1,38 @@
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import { Purchase } from 'calypso/lib/purchases/types';
+import PrePurchaseNotice from './prepurchase-notice';
+
+import './style.scss';
+
+type Props = {
+	purchase: Purchase;
+};
+
+const AkismetProductOverlapsOwnedProductNotice: FunctionComponent< Props > = ( { purchase } ) => {
+	const translate = useTranslate();
+	const subscriptionUrl = `/me/purchases/siteless.akismet.com/${ purchase.id }`;
+
+	const message = translate(
+		'The plan you are about to purchase contains all the same features of your current plan. Consider {{link}}removing your %(product)s subscription{{/link}} to avoid paying for both plans.',
+		{
+			comment: 'The `product` variable refers to the product the customer owns already.',
+			components: {
+				link: <a href={ subscriptionUrl } />,
+			},
+			args: {
+				product: purchase.productName,
+			},
+		}
+	);
+
+	return (
+		<PrePurchaseNotice
+			message={ message }
+			linkUrl={ subscriptionUrl }
+			linkText={ translate( 'Manage subscription' ) }
+		/>
+	);
+};
+
+export default AkismetProductOverlapsOwnedProductNotice;
```

Comment:
IMO this looks a bit iffy on mobile, the Manage Subscription link is very small. Maybe we could consult somebody on the design team to make sure this looks as good as we can make it?
![image](https://user-images.githubusercontent.com/65001528/228675138-15c82f75-9bc6-4fea-aa5c-309fa023cb0e.png)

Reply:
I see though that you aren't updating any styles, this is probably outside the scope of this diff